modules/custom/az_event/az_event_trellis/js/az_event_trellis_date.es6.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * @file
+ * Trellis date range picker.
+ */
+
+(($, Drupal, drupalSettings, once) => {
+  Drupal.behaviors.trellisDatePicker = {
+    attach(context) {
+      $(once('aztrellisdate', '.az-trellis-daterange', context))
+        // eslint-disable-next-line func-names
+        .each(function () {
+          const begin = this;
+          const id = $(this).data('az-trellis-daterange-end');
+          const end = $(`#${id}`).get(0);
+          // todo figure out how to make CSS declaration unnecessary.
+          // eslint-disable-next-line no-unused-vars, no-undef, new-cap
+          const picker = new easepick.create({
+            element: begin,
+            css: drupalSettings.trellisDatePicker.css,
+            zIndex: 10,
+            RangePlugin: {
+              elementEnd: end,
+            },
+            plugins: ['RangePlugin'],
+          });
+        });
+    },
+  };
+})(jQuery, Drupal, drupalSettings, once);
```

Comment:
Nitpick:  does this really need jQuery?

```suggestion
((Drupal, drupalSettings, once) => {
  Drupal.behaviors.trellisDatePicker = {
    attach(context) {
      const elements = once('aztrellisdate', '.az-trellis-daterange', context)
        elements.forEach(function (element) {
          const begin = element;
          const id = element.dataset.az-trellis-daterange-end;
          const end = document.getElementById(id);
          // todo figure out how to make CSS declaration unnecessary.
          // eslint-disable-next-line no-unused-vars, no-undef, new-cap
          const picker = new easepick.create({
            element: begin,
            css: drupalSettings.trellisDatePicker.css,
            zIndex: 10,
            RangePlugin: {
              elementEnd: end,
            },
            plugins: ['RangePlugin'],
          });
        });
    },
  };
})(Drupal, drupalSettings, once);
```

Reply:
Agreed. Thanks! I made a few eslint fixes and also removed the jquery library dependency declaration from the libraries.yml file.